enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -571,8 +573,17 @@ def write_connection_file(self):
             self.hb_port = ports[3]
             self.control_port = ports[4]
 
-            return super(RemoteKernelManager, self).write_connection_file()
+            cf = super(RemoteKernelManager, self).write_connection_file()
+            username=self.user_overrides.get('KERNEL_USERNAME',None)
```

Comment:
Is there a better way to get the username at this point? like:
`username = KernelSessionManager.get_kernel_username(**kwargs)`

It would be the best to write_connection_file to the correct user rather than chown-ing it later. But idk